nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentsMigrator.java

```diff
@@ -139,6 +251,36 @@ protected void migrateComments(CoreSession session, RelationManager relationMana
 
         Graph graph = relationManager.getGraph(config.graphName, session);
         graph.remove(statement);
+
+    }
+
+    /**
+     * @since 11.1
+     */
+    protected void stripACLs(DocumentModel comment) {
+        ACP acp = comment.getACP();
+        acp.removeACL(LOCAL_ACL);
+        comment.setACP(acp, true);
+    }
+
+    /**
+     * @since 11.1
+     */
+    protected void migrateCommentsFromPropertyToSecured(CoreSession session, PropertyCommentManager commentManager,
+            DocumentModel comment) {
+        comment.putContextData(DISABLE_NOTIFICATION_SERVICE, TRUE); // Remove notifications
+        session.move(comment.getRef(), commentManager.getAncestorRef(session, comment), null);
+        stripACLs(comment);
+    }
+
+    /**
+     * @since 11.1
+     */
+    protected String buildQuery(List<String> parentIds, String docType) {
+        String inClause = parentIds.stream().map(entry -> "'" + entry + "'").collect(Collectors.joining(", "));
```

Comment:
No need for `.map`, just do `.stream().collect(Collectors.joining("', '", "('", "')"))`